octobot/config/config_schema.json

```diff
@@ -16,6 +16,15 @@
     "profile": {
       "type": "string"
     },
+    "community-device_id": {
+      "type": "string"
+    },
+    "community-token": {
+      "type": "string"
+    },
+    "community_environment": {
+      "type": "string"
+    },
```

Comment:
Should we create a root key `community` that contains `token`, `environment` and other related keys ?